docs/system/hardware-metrics.md

```diff
@@ -10,11 +10,14 @@ This document describes instruments and attributes for common hardware level
 metrics in OpenTelemetry. Consider the [general metric semantic conventions](/docs/general/metrics.md#general-metric-semantic-conventions)
 when creating instruments not explicitly defined in the specification.
 
+This document is being converted to specific hardware metrics, parts of this document that have already been
+converted are now located in the [Hardware] (/docs/hardware/README.md) folder and are no longer present in this file.
+
+Please note that this is an ongoing process and may take some time to complete.
```

Comment:
do we have an issue that outlines what needs to happen? Can we add a todo comment here and add link to that issue?

Reply:
I have created a tracking issue for this https://github.com/open-telemetry/semantic-conventions/issues/1309